eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -0,0 +1,83 @@
+param(
```

Comment:
This script appears to be intended to be called from a command line and not really imported/called as part of another script.

Can we consider wrapping things into a function to make calling it more explicit?  Something like:

```powershell
param(
    [string]$ArtifactsPath
)
. (Join-Path $PSScriptRoot ../../common/scripts/common.ps1)
$ErrorActionPreference = 'Continue'

function Get-SmokeTestPkgProperties
{
    Param
    (
        [string]$ArtifactsPath
    )

    // ... CODE ..
}

if ($ArtifactsPath -ne $null)
{
    Get-SmokeTestPkgProperties($ArtifactsPath)
}
```

Reply:
Could we also please disable writing to host by default and add a param to enable it?   The output from calling this overwhelms the output from callers and makes it quite difficult to follow the flow.

Reply:
We can use Write-Verbose so we can turn it on only for debugging as necessary.

Reply:
Yes, please use `Write-Verbose` or `Write-Debug` and not invent another message writing system for PowerShell that designed a consistent one.